tests/test_ingest.py

```diff
@@ -0,0 +1,41 @@
+import pathlib
+
+from . import *
```

Comment:
we typically avoid `import *` - 
* some linter tools cant pin down imported symbols as used/unused
* reader doesn't know where some symbols come from (esp. if multiple * used)
* risk of overriding/colliding silently if upstream dependencies shift.

Reply:
agreed, not a big fan of `import *` either. I had it there for quick prototyping then forgot to change. PR updated now for the fix

Reply:
sounds good - thought you tended not to, so apologies if preachy/condescending sounding; just saving potential for back & forth :+1: